ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -679,6 +765,7 @@ fi
 1. First time provisioning of IPU system(where MAC gets set in node policy)
 2. Upgrade-for any update to P4 package.
 3. Upgrade-for node policy. Other changes in node policy thro load_custom_pkg.sh.
+4. Check if file-> post_init_app.sh exists.
```

Comment:
I just realized this. Can we add a comment here or somewhere that today we take over full ownership of the 2 files `load_custom_pkg.sh` and `post_init_app.sh` and that these files cannot be used by customers to do anything extra that they want. If they have any additional needs, then we would need to modify code to append our contents to these files rather than taking over them completely.